index.html

```diff
@@ -474,89 +481,108 @@ <h3>MediaStreamTrack frame stats</h3>
-              </li>
-            </ol>
           </ol>
         </dd>
       </dl>
     </section>
     <section>
-      <h4>The MediaTrackFrameStats dictionary</h4>
+      <h4>The MediaStreamTrackVideoStats interface</h4>
       <div>
-        <pre class="idl" data-cite="HR-TIME"
->dictionary MediaTrackFrameStats {
-  unsigned long long deliveredFrames;
-  unsigned long long discardedFrames;
-  unsigned long long totalFrames;
+        <pre class="idl">[Exposed=Window]
+interface MediaStreamTrackVideoStats {
+  readonly attribute unsigned long long deliveredFrames;
+  readonly attribute unsigned long long discardedFrames;
+  readonly attribute unsigned long long totalFrames;
 };
-</pre>
+        </pre>
+        <div>
+          <p>Let the {{MediaStreamTrackVideoStats}} have internal slots
+          <dfn class=export data-dfn-for="MediaStreamTrackVideoStats">[[\DeliveredFrames]]</dfn>,
+          <dfn class=export data-dfn-for="MediaStreamTrackVideoStats">[[\DiscardedFrames]]</dfn>
+          and
+          <dfn class=export data-dfn-for="MediaStreamTrackVideoStats">[[\TotalFrames]]</dfn>,
+          initialized to 0.
+          </p>
+        </div>
         <section>
-          <h4>Dictionary {{MediaTrackFrameStats}} Members</h4>
-          <dl data-link-for="MediaTrackFrameStats"
-          data-dfn-for="MediaTrackFrameStats" class="dictionary-members">
+          <h4>Attributes</h4>
+          <dl data-link-for="MediaStreamTrackVideoStats"
+          data-dfn-for="MediaStreamTrackVideoStats" class="attributes">
             <dt>
               <dfn data-idl="">deliveredFrames</dfn> of type <span class=
-              "idlMemberType">unsigned long long</span>
+              "idlMemberType">unsigned long long, readonly</span>
             </dt>
             <dd>
               <p>
-                The total number of [= delivered frames =].
+                Upon getting, return this track stats'
+                {{MediaStreamTrackVideoStats/[[DeliveredFrames]]}}.
               </p>
             </dd>
             <dt>
               <dfn data-idl="">discardedFrames</dfn> of type <span class=
-              "idlMemberType">unsigned long long</span>
+              "idlMemberType">unsigned long long, readonly</span>
             </dt>
             <dd>
               <p>
-                The total number of [= discarded frames =].
+                Upon getting, return this track stats'
+                {{MediaStreamTrackVideoStats/[[DiscardedFrames]]}}.
               </p>
             </dd>
             <dt>
               <dfn data-idl="">totalFrames</dfn> of type <span class=
-              "idlMemberType">unsigned long long</span>
+              "idlMemberType">unsigned long long, readonly</span>
             </dt>
             <dd>
               <p>
-                The [= total frames =] count.
+                Upon getting, return this track stats'
+                {{MediaStreamTrackVideoStats/[[TotalFrames]]}}.
```

Comment:
can the definitions in line 450 onwards be moved to here... because the explanations here feel incomplete and you've to search for the definitions which are not linked.

Reply:
The getter returns the internal slot (which is cached), not `[= total frames =]` directly. They underlying counters are referenced by the `videoStats` getter so I thought it made sense to have the definition of these earlier in the document. But alternatively I could move them to the `MediaStreamTrackVideoStats interface` section?

Reply:
Nvm I can try to move it as you suggested, there's nothing preventing mixing definitions and getters